CONTRIBUTING.markdown

```diff
@@ -35,7 +35,7 @@ merge using fast-forward and push by hand. This will avoid creating a merge
 commit (which makes very little sense for a one-commit pull request).
 
     $ git merge --ff-only feature-branch
-    $ python manage.py test
+    $ python manage.py test --settings=pdc.settings_test
```

Comment:
Could this be handled in code instead? I often forgot that there needs to be extra parameter.